packages/medusa-payment-adyen/README.md

```diff
@@ -0,0 +1,77 @@
+# Adyen
```

Comment:
todo: This is not actively maintained. I don't think we should deprecate, but add a notice stating that this is not ready for production use. And that community contributions are highly appreciated.